### ISSUE TITLE
Feature request: Introductions

### ISSUE THREAD

**Issue by baisong** [monicahq/monica#560]
I love MonicaHQ!

Searched the issues and didn't see this idea, wanted to drop it in here.

I'd love a special type of record for "Introductions," alongside phone calls, activities, gifts, etc.

Example:
> I met my friend Ahmad through our mutual friend Bozhang, who set us up to meet over email.
> Now, I want to record this happened in MonicaHQ, on Bozhang's profile page.

I'm thinking the "fields" on the record could be as basic as:

1. Introduced contact (choose from MonicaHQ contacts) (requred)
2. Optional date of introduction
3. Optional method (i.e. email, meeting, event, other)
4. Optional note.

Perhaps the introduced contact could inherit this reference as well. Now when I look at Ahmad's profile page, I see "Met through Bozhang" with a link to Bozhang's profile page.

**Comment by djaiss** [comment]
Interesting idea! Definitely something we can investigate!

**Comment by hupili** [comment]
Nice feature! How about rewording the title to be "Feature request: referral chain"?

First thought of "Introduction" is like introducing the function of the website, like a getting-started page.

**Comment by djaiss** [comment]
This feature is done and in production.
